minutes/2020-03-24.md

```diff
@@ -0,0 +1,92 @@
+# Jakarta EE Platform Call
+
+Date: 2020-03-24
+
+Present:
+
+- Kevin Sutter (IBM)
+- Scott Marlow (Red Hat)
+- Ivar Grimstad (Eclipse Foundation)
+- Tanja Obradovic (Eclipse Foundation)
+- Jonathan Gallimore (Tomitribe)
+- Dalia Abo Sheasha (IBM)
+- Jesse McConnell (Webtide)
+- Ed Bratt (Oracle)
+- Bill Shannon (Oracle)
+- Paul Nicolucci (IBM)
+- Jonathan Coustick (Payara)
+- Steve Millidge (Payara)
+- Werner Keil (Committer)
+- Andy McCright (IBM)
+- Martin Stefanko (Red Hat)
+- Edwin Amoakwa
+- Carlos Andres De La Rosa(Eclipse committer)
+- Nathan Erwin (Eclipse Contributor)
+- Scott Kurz (IBM)
+- Emily Jiang (IBM)
+- Scott Stark (Red Hat)
+- BJ Hargrave (IBM)
+
+## Agenda
+
+### [IG, BS] Suggestions for Call Logistics
+
+- Always have the call (even with empty agenda upfront)
+- Agendas are everyone’s responsibility.  Anyone can add agenda items to the list -- just indicate who originated the agenda item with name or initials.
+- Meeting reminder (at least 24 hours before call)
+- Start Zoom early (1 hour) and display banner with start time
+   - difficult to accomplish since we can only have one zoom call active at any one time.  Or, other meetings may conflict.
+   - FYI, next week’s call puts us back in sync (due to daylight savings time difference between US and Europe)
+- If meeting ends early, leave Zoom session running with banner displaying that information
+   - This should be doable
+- Everyone should add their own name to the agenda/minutes as attending the call
+- Everyone please help keep minutes :)
+- Last weeks, unmeeting was recorded via Bill Shannon keeping minutes, we should do that for future unofficial calls.
```

Comment:
Nit...  Should be "Last week's unmeeting was...".  Change it if you wish.  The intent is still there.